lib/x509.mli

```diff
@@ -490,6 +494,10 @@ module Certificate : sig
   (** [pp ppf cert] pretty-prints the certificate. *)
   val pp : t Fmt.t
 
+  (** [pp' pp_custom_extensions ppf cert] pretty-prints the certificate using
+      [pp_custom_extensions] for [Extension.Unsupported _] extension. *)
+  val pp' : (Asn.oid * string) Fmt.t -> t Fmt.t
```

Comment:
I wonder whether we can make this composable? But I guess we'd need to use extensible variants for this? It'd be nice if e.g. webauthn (or albatross) could inject their pretty-printer for their extension...

Reply:
I thought of having an argument `Asn.oid -> (Asn.oid * string) Fmt.t option` and then you apply a list of functions or something until you have a `Some pp` or default to a "unknown extension" printer. But in the end I didn't come up with a nice solution, and this is simple.